.github/workflows/build-packages.yml

```
@@ -3,13 +3,29 @@ name: Build and Release Packages
+      - "v*"
   workflow_dispatch:
     inputs:
       version:
-        description: 'Version to build (leave empty for auto from meson.build)'
+        description: "Version to build (leave empty for auto from meson.build)"
         required: false
         type: string
+      release_body:
+        description: "Release body (leave empty to use CHANGELOG.md or generate from git commits)"
+        required: false
+        type: textarea
```

actionlint (1.7.7)

input type of workflow_dispatch event must be one of "string", "number", "boolean", "choice", "environment" but got "textarea"

(syntax-check)